source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js

```diff
@@ -95,6 +98,18 @@
         };
     }
 
+    CalificatorsManager.prototype.toEvaluators = function () {
+        var calificators = _.map(this.calificators(), function (e) {
+            return e.toJs();
```

Comment:
I think that it is not necessary to convert calificators to JSON objects, maybe it is enough reading the properties.